### DISCUSSION TITLE
Compatibile with MacOS/Intel?

### DISCUSSION THREAD

**Discussion by forrie** [taviso/123elf#124] [General]
Apple has a different environment, but we have Macports for most of the GNU and other stuff.   I wonder if there is a way to get this working under (at least) MacOS Monterey.  Has anyone mentioned hacking this?

**Comment by rruhle** [comment]
Not familiar with the Mac, but if Macports wont run .deb files, a Mac developer may need to work with the source code for a compatible compile.
